src/api_server/src/parsed_request.rs

```diff
@@ -717,6 +719,19 @@ pub(crate) mod tests {
         assert!(ParsedRequest::try_from_request(&req).is_ok());
     }
 
+    #[test]
+    fn test_try_from_put_entropy() {
+        let (mut sender, receiver) = UnixStream::pair().unwrap();
+        let mut connection = HttpConnection::new(receiver);
+        let body = "{}";
+        sender
+            .write_all(http_request("PUT", "/entropy", Some(body)).as_bytes())
+            .unwrap();
+        assert!(connection.try_read().is_ok());
+        let req = connection.pop_parsed_request().unwrap();
+        assert!(ParsedRequest::try_from_request(&req).is_ok());
```

Comment:
Nit it is usually better to match the specific value.
```suggestion
        assert_eq!(ParsedRequest::try_from_request(&req), ParsedRequest { .. });
```
or as close as possible e.g.
```
assert!(matches!(ParsedRequest::try_from_request(&req), Ok(...)))
```

Reply:
I think that finding what `ParsedRequest` should look like is not quite straight-forward and I don't think it's worth the effort here.

Reply:
Okay.